### PR TITLE
Change window access modifier to open

### PR DESCRIPTION
**Pull request by julianrex**
Hello friends 😄 

Small PR to change the window to open, along with a few other changes noted as comments.

Tested manually in an iPadOS application.

/cc @OdNairy

### CODE REVIEW

File: Sources/Fingertips/FingerTipWindow.swift [hunk 1]
```diff
@@ -200,13 +202,6 @@ public class FingerTipWindow: UIWindow {
         perform(#selector(removeInactiveFingerTips), with: nil, afterDelay: 0.1)
     }
 
-    func cancelScheduledFingerTipRemoval() {
-        fingerTipRemovalScheduled = true
-        NSObject.cancelPreviousPerformRequests(withTarget: self,
-                                               selector: #selector(removeInactiveFingerTips),
-                                               object: nil)
-    }
-
```
Comment (julianrex) [inline Sources/Fingertips/FingerTipWindow.swift:209]: This is not used; I imagine that it was once used in conjunction with the `perform(#selector(removeInactiveFingerTips), with: nil, afterDelay: 0.1)`. I do not see a need for it, but perhaps I'm not seeing the full picture.

File: Sources/Fingertips/FingerTipWindow.swift [hunk 2]
```diff
@@ -233,16 +228,27 @@ public class FingerTipWindow: UIWindow {
             return
         }
 
-        UIView.animate(withDuration: fadeDuration) {
+        let animation = {
```
Comment (julianrex) [inline Sources/Fingertips/FingerTipWindow.swift:231]: The `animated` parameter wasn't being observed. As it is it seems that `animated` is always true, so we could simplify and rename to `removeFingerTipAnimated(with:)`.

**Comment by julianrex** [comment]
https://github.com/mapbox/Fingertips/blob/aca10de447b478e65bd31a1b0ab10b5f7886eee7/Sources/Fingertips/FingerTipWindow.swift#L87

This causes the following warning in the console:

```
[Window] Manually adding the rootViewController's view to the view hierarchy is no longer supported. Please allow UIWindow to add the rootViewController's view to the view hierarchy itself.
```

File: Sources/Fingertips/FingerTipWindow.swift [hunk 3]
```diff
@@ -11,20 +11,10 @@ class FingerTipView: UIImageView {
...
-    override var rootViewController: UIViewController? {
-        set {
-            super.rootViewController = newValue
-        }
-
-        get {
-            return FingerTipWindow.fingerTipWindow?.rootViewController ?? super.rootViewController
-        }
-    }
-}
```
Comment (julianrex) [inline Sources/Fingertips/FingerTipWindow.swift:24]: This was causing a warning in logs; the getter also looks suspicious - sharing a view controller between UIWindows sounds dangerous. I don't see any issue with removing this code.

**Review by OdNairy — APPROVED** [review]
Thank you @julianrex for the improvements!

**Pull request merged**